sdk/python/kfp/components/yaml_component.py

```diff
@@ -1,4 +1,6 @@
-# Copyright 2021 The Kubeflow Authors
+"""Functions for loading components from compiled YAML."""
+
+# Copyright 2021-2022 The Kubeflow Authors
```

Comment:
nit: I think copyright should appear first, can't find a reference though.

Reply:
Fixed 👍🏻